packages/dd-trace/test/llmobs/plugins/langchain/index.spec.js

```diff
@@ -59,6 +63,13 @@ describe('integrations', () => {
 
   describe('langchain', () => {
     before(async () => {
+      iastFilter.isDdTrace = file => {
```

Comment:
Could this somehow be done outside of individual tests? Maybe in setup files?

Reply:
The more I think about it the more I wonder if this could be done another way entirely instead too.

Reply:
We can (and probably should) encapsulate this in a test helper, but I think that can probably be done in a future PR once it becomes necessary, i.e. when we have multiple instrumentations. WDYT?